lib/csv/test_csv.nit

```diff
@@ -1,201 +0,0 @@
-# This file is part of NIT ( http://www.nitlanguage.org ).
-#
-# This file is free software, which comes along with NIT. This software is
-# distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
-# without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
-# PARTICULAR PURPOSE. You can modify it is you want, provided this header
-# is kept unaltered, and a notification of the changes is added.
-# You are allowed to redistribute it and sell it, alone or is a part of
-# another product.
-
-# Tests for `csv`.
-module test_csv is test_suite
```

Comment:
Nope, it's just that I did not care not want to maintain this test_suite since I was basically re-writing the module.

Besides test suites could use some love to simplify them (or maybe test suites like this one are heavily written such that they are essentially unreadable).